docs/apps/clearml_session.md

```diff
@@ -11,7 +11,33 @@ in the UI and send it for long-term training on a remote machine.
 
 **If you are not that lucky**, this section is for you :)
 
-## What Does ClearML Session Do?
+ClearML provides tools that allow you to launch remote sessions and to execute code on a remote machine that better 
+meets resource needs:
+* [GUI Applications](#gui-applications) (available under ClearML Enterprise Plan)
+  * [JupyterLab](../webapp/applications/apps_jupyter_lab.md) - Launch a JupyterLab session on a remote machine 
+  * [VS Code](../webapp/applications/apps_vscode.md) - Launch a VS Code session on a remote machine 
+* [Remote PyCharm](#remote-pycharm) - Work with PyCharm in a remote session over SSH
+* [Clearml Session CLI](#clearml-session-cli) - Launch a JupyterLab and VS Code session on a remote machine
+
+
+## GUI Applications
+
+:::important Enterprise Feature
+This feature is available under the ClearML Enterprise plan
+:::
+
+The ClearML Enterprise Server provides GUI applications for setting up remote sessions in VS Code and JupyterLab. These
+apps provide local links to access the JupyterLab or VS Code on a remote machine over a secure and encrypted SSH connection,
+letting you use the IDE as if you're running on the target machine itself
+
+For more information, see [JupyterLab](../webapp/applications/apps_jupyter_lab.md) and/or [VS Code](../webapp/applications/apps_vscode.md).
+
+## Remote PyCharm
```

Comment:
@pollfly The pycharm plugin is not the same capability as clearml session - Remove from remote session options, and keep it as a note as was before.